mypy/checker.py

```diff
@@ -7450,13 +7450,24 @@ def infer_operator_assignment_method(typ: Type, operator: str) -> tuple[bool, st
     For example, if operator is '+', return (True, '__iadd__') or (False, '__add__')
     depending on which method is supported by the type.
     """
+
+    def find_method(inst: Instance, method: str) -> str | None:
```

Comment:
Does mypyc deal well with nested functions now? Seems pretty easy to refactor to avoid the nested function.